addons/auth_totp_portal/static/src/js/totp_frontend.js

```diff
@@ -30,6 +30,52 @@ function fromField(f, record) {
         code.maxLength = 6;
         code.minLength = 6;
         return code;
+    case 'secret':
```

Comment:
:exploding_head:

Reply:
Well, I'll add a note on why I do that. 
I could eventually reduce a bit by using `$('<div/>, properties...)` instead of `document.createElement`
The idea is to copy the copyclipboard widget structure in the frontend usage of the wizard (among the other crapy bricolage already done to make this wizard work in frontend, except that for secret, we need a lot more elements to add..)

Reply:
Well, I don't know if you prefer this (see "compact copyclipboard code" commit) even it if stays a bricolage anyway.

Reply:
Ok ^^